pom.xml

```diff
@@ -102,28 +101,16 @@
     <version.jakartaee-api>9.1-M2</version.jakartaee-api>
     <version.tomee-patch-plugin>0.9</version.tomee-patch-plugin>
 
-    <!--
-      JavaMail is both API and IMPL so we don't have them in the jakartaee-api uber jar.
-      We decided to add them here in the project so we can patch/update the 2 libraries without having
```

Comment:
Guess we should keep these comments

Reply:
Moving away from Geronimo API + Impls need to be discussed on the list. 

I wouldn't change that in this PR. We should keep it as ist and do the cleanup/renaming. If we have consensus on the list, we might switch away.

Reply:
to be clear my intent is to keep Geronimo Impl. and swtich only the API.
but i get your point and will discuss that on the mailing list.

Reply:
There is also an ongoing discussion regarding Geronimo components on their list. There might be changes in the near future (attic, etc), but that discussion hasn't ended yet. So we might need to discuss / change things in the future.